query-service-impl/src/test/java/org/hypertrace/core/query/service/pinot/converters/PinotFunctionConverterTest.java

```diff
@@ -211,7 +211,7 @@ void convertAvgRateFunction() {
         .thenReturn(Optional.of(Duration.ofSeconds(10)));
 
     assertEquals(
-        "SUM(DIV(foo, 2.0))",
+        "SUM(foo / 2.0)",
```

Comment:
Just curious: Do we have any value level test that compares prior (SUM(DIV(foo, 2.0)) vs SUM(foo / 2.0, resulting in the same values?

Reply:
That would be testing pinot's functionality - that `DIV` and `/` both actually... divide. Not sure it's worth writing a test, but we can verify by hand real quick.

Reply:
```
select sum(num_calls), sum(num_calls / 60.0), sum(div(num_calls, 60.0)) ...
```

![image](https://user-images.githubusercontent.com/45047841/175542557-68931067-206e-468d-8e2e-741e90b0dbd8.png)